src/praisonai/tests/unit/bots/test_server_retry_after.py

```
@@ -0,0 +1,231 @@
+
+    async def run():
+        return await _delivery.deliver_with_retry(
+            Adapter(),
+            "chanX",
+            "hi",
+            max_attempts=3,
+            platform="discord",
+            rate_limiter=limiter,
+        )
+
+    success, err = asyncio.run(run())
```

Ruff (0.15.18)

[warning] Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)